scripts/patterns-reorg/page_content.py

```diff
@@ -195,39 +197,29 @@ def index_page_content() -> str:
 
 # Introduction to Qiskit
 
-[INSERT BIG PATTERNS AND TOOLS IMAGE]
-
 
 The name "Qiskit" is a general term referring to a collection of software for executing programs on quantum computers. Most notably among these software tools is the open-source Qiskit SDK, and the runtime environment (accessed using Qiskit Runtime) through which you can execute workloads on IBM&reg; quantum computers. As quantum technology evolves, so does Qiskit, with new capabilities released every year which expand this core collection of quantum software. 
 
 In addition, there are many open-source projects that are part of the broader Qiskit ecosystem. These software tools are not part of Qiskit itself, but rather interface with Qiskit and can provide valuable additional functionality.
 
+![../\_images/patterns.svg](/images/patterns.svg)
+
 
 ## The Qiskit SDK
 
 The Qiskit SDK (package name [`qiskit`](https://pypi.org/project/qiskit/)) is an open-source SDK for working with quantum computers at the level of extended (static, dynamic, and scheduled) quantum circuits, operators, and primitives. This library is the core component of Qiskit; it is the largest package under the Qiskit name with the broadest suite of tools for quantum computation, and many other components interface with it.
 
 Some of the most useful features of the Qiskit SDK include:
 
-### Circuit building tools 
-([`qiskit.circuit`](/api/qiskit/circuit))
-For initializing and manipulating registers, circuits, instructions, gates, parameters, and control flow objects. 
+- Circuit building tools ([`qiskit.circuit`](/api/qiskit/circuit)) - For initializing and manipulating registers, circuits, instructions, gates, parameters, and control flow objects. 
 
-### Circuit library 
-([`qiskit.circuit.library`](/api/qiskit/circuit_library)) 
-A vast range of circuits, instructions, and gates - key building blocks for circuit-based quantum computations.
+- Circuit library ([`qiskit.circuit.library`](/api/qiskit/circuit_library)) - A vast range of circuits, instructions, and gates - key building blocks for circuit-based quantum computations.
 
-### Quantum Information
-([`qiskit.quantum_info`](/api/qiskit/quantum_info))
-A toolkit for working with quantum states, operators and channels, using exact calculations (no sampling noise). Use this module to specify input observables and analyse fidelity of outputs from primitives queries.
+- Quantum Information ([`qiskit.quantum_info`](/api/qiskit/quantum_info)) - A toolkit for working with quantum states, operators and channels, using exact calculations (no sampling noise). Use this module to specify input observables and analyse fidelity of outputs from primitives queries.
```

Comment:
```suggestion
- Quantum information ([`qiskit.quantum_info`](/api/qiskit/quantum_info)) - A toolkit for working with quantum states, operators, and channels, using exact calculations (no sampling noise). Use this module to specify input observables and analyze fidelity of outputs from primitives queries.
```

Reply:
Not sure if the "I" in Information should be capitalized as part of a package name or not - suggesting the change here since we try to keep things lower-case as much as possible, but feel free to change it back